src/store/mutations.ts

```diff
@@ -394,8 +394,8 @@ export const setNrResponse = (state: StateIF, nr: NameRequestI) => {
   }
 }
 
-export const updateReservationNames = (state: StateIF, nrName: [] = []) => {
-  nrName.forEach(({ choice, name = '', designation = '' }) => {
+export const updateReservationNames = (state: StateIF, nrNames: [] = []) => {
+  nrNames.forEach(({ choice, name = '', designation = '' }) => {
```

Comment:
I like when array variables are plural, and then you can iterate over their singular elements :)